tests/h/search/query_test.py

```diff
@@ -89,12 +89,25 @@ def search(self, search, pyramid_request):
 
 
 class TestGroupFilter(object):
+    def test_matches_only_annotations_from_specified_group(self, search, Annotation, group):
+        Annotation(groupid='group2')
+        Annotation(groupid='group3')
+        group1_annotations = [Annotation(groupid=group.name).id,
+                              Annotation(groupid=group.name).id]
+
+        result = search.run({'group': group.pubid})
+
+        assert set(result.annotation_ids) == set(group1_annotations)
```

Comment:
This should be `sorted(...) == sorted(..)` as in the other tests, no?

Reply:
Oops! I implemented this branch/test a long time ago before we changed to sorted-forgot to even check for that! Good catch!